test/amazonProductApiTest.coffee

```diff
@@ -104,9 +103,8 @@ describe 'client.itemLookup(query, cb)', ->
 
     describe 'when callback is passed', ->
       it 'should return search results from amazon', (done) ->
-        client.itemLookup query, (err, results, response) ->
-          results.should.be.an.Array
-          response.should.be.an.Array
+        client.itemLookup query, (err, results) ->
+          results.should.be.an.Object
```

Comment:
Would be nice to check if the object contains the required properties and also check the property types.